src/amo/components/HomeHeroBanner/index.js

```diff
@@ -22,57 +22,58 @@ export class HomeHeroBannerBase extends React.Component<Props> {
       (
         <HeroSection
           key="hero-1"
-          linkTo="/addon/temporary-containers/"
+          linkTo="/addon/facebook-video-downloader-hd/"
         >
-          <h3>{i18n.gettext('Temporary Containers')}</h3>
+          <h3>{i18n.gettext('Facebook Video Downloader')}</h3>
 
           <p>
-            {i18n.gettext('Open pages with disposable data containers')}
+            {i18n.gettext('Download with a single click')}
           </p>
         </HeroSection>
       ),
       (
         <HeroSection
           key="hero-2"
-          linkTo="/addon/momentumdash/"
+          linkTo="/addon/autoformer/"
         >
-          <h3>{i18n.gettext('Momentum')}</h3>
+          <h3>{i18n.gettext('AutoFormer+')}</h3>
 
           <p>
-            {i18n.gettext(`Replace your new tab with a personal
-              dashboard—to-do lists, weather forecasts and more`)}
+            {i18n.gettext(`Fill out a lot of forms? This might help`)}
```

Comment:
This doesn't need to be a backtick-quoted string anymore, surprised eslint doesn't complain! I guess it's fine though.

Reply:
I don't think eslint cares about this, but good point. It must have been a longer string before and I neglected to replace the backticks with single quotes. I'll change it.